sql/core/src/test/scala/org/apache/spark/sql/StreamTest.scala

```diff
@@ -75,6 +76,8 @@ trait StreamTest extends QueryTest with Timeouts {
   /** How long to wait for an active stream to catch up when checking a result. */
   val streamingTimeout = 10.seconds
 
+  val outputMode: OutputMode = Append
```

Comment:
We might want to make this an argument to `streamTest` with a default.  I think in a single suite we might want to be able to have more than one kind of test.